service/messageHandler.go

```diff
@@ -73,18 +79,30 @@ func (mapper *AnnotationMapperService) HandleMessage(msg kafka.FTMessage) error
 
 	marshalledAnnotations, err := json.Marshal(mappedAnnotations)
 	if err != nil {
-		requestLog.Error("Error marshalling the concept annotations", err)
+		log.WithMonitoringEvent(mapperEvent, tid, annotationsContentType).
+			WithUUID(metadataPublishEvent.UUID).
+			WithValidFlag(true).
+			WithError(err).
```

Comment:
what does the isValid refer to here?

Reply:
It is explained [here](https://github.com/Financial-Times/go-logger/pull/5/files#diff-04c6e90faac2675aa89e2176d2eec7d8R26).
It says if the processed message is valid.